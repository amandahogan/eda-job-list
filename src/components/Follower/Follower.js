import React, { PropTypes } from 'react'
import { Image } from 'semantic-ui-react'

class Follower extends React.Component {
  render () {
    return (
      <Image floated='left' size='mini' src={this.props.avatar} />
    )
  }
}

Follower.propTypes = {
  avatar: PropTypes.string.isRequired
}

export default Follower
