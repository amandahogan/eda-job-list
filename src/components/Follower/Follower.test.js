import { shallow } from 'enzyme'
import React from 'react'

import Follower from './Follower'

const avatar = ''

describe('<Follower />', () => {
  it('matches snapshot', () => {
    expect(
      shallow(<Follower avatar={avatar} />).getNode()
    ).toMatchSnapshot()
  })
})
