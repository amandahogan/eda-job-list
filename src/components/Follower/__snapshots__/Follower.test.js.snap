exports[`<Follower /> matches snapshot 1`] = `
<Image
  as="img"
  floated="left"
  size="mini"
  src=""
  ui={true} />
`;
